milli/tests/search/query_criteria.rs

```diff
@@ -373,7 +373,7 @@ fn criteria_mixup() {
 fn criteria_ascdesc() {
     let path = tempfile::tempdir().unwrap();
     let mut options = EnvOpenOptions::new();
-    options.map_size(10 * 1024 * 1024); // 10 MB
+    options.map_size(12 * 1024 * 1024); // 10 MB
```

Comment:
Ok, so here, we don't understand why but we need this extra 1-2MiB, right?

Reply:
yeah, as discussed, I don't really know why that is, but this fixes the issue 🤔